.circleci/config.yml

```diff
@@ -0,0 +1,108 @@
+version: 2.1
+jobs:
+  lint:
+    docker:
+      - image: salsify/ruby_ci:2.5.8
```

Comment:
why use the 2.5.8 image instead of the 2.6.x one?

Reply:
It shouldn't really matter since we lint with a target Ruby version of our last supported Ruby version but I thought running Rubocop using the same target Ruby version would reinforce that approach.